prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -216,6 +216,49 @@ periodics:
     - name: github
       secret:
         secretName: oauth-token
+- cron: "15 15-23 * * 1-5"  # Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
+  name: ci-oss-test-infra-autobump-knative-prow
+  cluster: test-infra-trusted
+  decorate: true
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: oss-test-infra
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
+    testgrid-tab-name: autobump-knative-prow
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/autobump:v20200622-168a90f1b0
+      command:
+      - /autobump.sh
+      args:
+      - /etc/github-token/oauth
+      - "Google OSS Prow Robot"
+      - google-oss-robot@google.com
+      volumeMounts:
+      - name: github
+        mountPath: /etc/github-token
+        readOnly: true
+      env:
+      - name: GH_ORG
+        value: GoogleCloudPlatform
+      - name: GH_REPO
+        value: oss-test-infra
+      - name: PLANK_DEPLOYMENT_FILE
+        value: prow/knative/cluster/400-plank.yaml
+      - name: COMPONENT_FILE_DIR
+        value: prow/knative/cluster
+      - name: CONFIG_PATH
+        value: prow/knative/cluster/400-plank.yaml # Trick autobump as this is required
+      - name: JOB_CONFIG_PATH
+        value: prow/knative/cluster/400-plank.yaml # Ditto trick
```

Comment:
`JOB_CONFIG_PATH` shouldn't be required. We should fix that upstream, that requirement is an oversight. I intentionally avoided assuming that `JOB_CONFIG_PATH` would be specified: https://github.com/kubernetes/test-infra/blob/d2cf7b0d1b296739c21b86a764cf36e32dbdf546/prow/cmd/autobump/bump.sh#L73

I'll make a quick PR for that.

Reply:
Sg, and thanks a lot

Reply:
https://github.com/kubernetes/test-infra/pull/18106